src/Services/Comments.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Moderation\Services;

```

Comment:
@arfenis you are inside a package you don't need to define service , you should just define the class outside